governance/program/src/instruction.rs

```diff
@@ -94,23 +103,30 @@ pub enum GovernanceInstruction {
         amount: u64,
     },
 
-    /// Withdraws governing tokens (Community or Council) from Governance Realm and downgrades your voter weight within the Realm
-    /// Note: It's only possible to withdraw tokens if the Voter doesn't have any outstanding active votes
-    /// If there are any outstanding votes then they must be relinquished before tokens could be withdrawn
+    /// Withdraws governing tokens (Community or Council) from Governance Realm
+    /// and downgrades your voter weight within the Realm Note: It's only
+    /// possible to withdraw tokens if the Voter doesn't have any outstanding
+    /// active votes If there are any outstanding votes then they must be
+    /// relinquished before tokens could be withdrawn
```

Comment:
Need some more newlines, not exactly like this, but you get the idea
```suggestion
    /// and downgrades your voter weight within the Realm.
    /// Note: It's only
    /// possible to withdraw tokens if the Voter doesn't have any outstanding
    /// active votes.
    /// If there are any outstanding votes then they must be
    /// relinquished before tokens could be withdrawn
```